plat/nvidia/tegra/include/t186/tegra186_private.h

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2017, ARM Limited and Contributors. All rights reserved.
```

Comment:
You can use Nvidia in the copyright notices.

Reply:
@antonio-nino-diaz-arm 

Just to be clear, you expect NVIDIA and ARM license in such cases, right?

Reply:
If you are adding new files you can just have NVIDIA and leave out Arm.